README.md

```diff
@@ -892,7 +892,8 @@ var log = bunyan.createLogger({
         type: 'rotating-file',
         path: '/var/log/foo.log',
         period: '1d',   // daily rotation
-        count: 3        // keep 3 back copies
+        count: 3,        // keep 3 back copies,
+        maxSizeKB: 1024 // rotate every 1024KB
```

Comment:
I think you should add a proper explanation of what the "maxSizeKB" attribute does.
If you expand the code below, you'll see:

"Currently, there is no support for providing a template for the rotated files, or for rotating when the log reaches a threshold size."

Which is no longer true.